tests/v2/validation/provisioning/config.go

```diff
@@ -5,10 +5,13 @@ import (
 	nodepools "github.com/rancher/rancher/tests/framework/extensions/rke1/nodepools"
 )
 
+type Version string
+
 const (
-	namespace               = "fleet-default"
-	defaultRandStringLength = 5
-	ConfigurationFileKey    = "provisioningInput"
+	namespace                       = "fleet-default"
+	defaultRandStringLength         = 5
+	ConfigurationFileKey            = "provisioningInput"
+	HardenedKubeVersion     Version = "v1.25.0"
```

Comment:
I think this name may be misleading. Since this is the version we don't support, not the highest version we support as the name would imply. But I can't think of a name that would reflect that. So maybe just set the default to `v1.24.99`?